bindings/wasm/examples/manifold-gltf.js

```diff
@@ -0,0 +1,176 @@
+// Copyright 2023 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Extension, ExtensionProperty, PropertyType} from '@gltf-transform/core';
+
+const NAME = 'EXT_manifold';
+
+export class ManifoldPrimitive extends ExtensionProperty {
```

Comment:
This file is designed to make it easy to read and write the EXT_manifold glTF extension regardless of whether someone is using Manifold.